puppet/manifests/default.pp

```diff
@@ -30,8 +79,27 @@
   }
 }
 
+# This needs to be on exactly two nodes.
+class gluster_cluster {
+  include glusterfs_common
+
+  gluster_peer { ['controller.seed-stack.local', 'worker.seed-stack.local']: }
```

Comment:
Oops, we autorequire the service, not the package. That means this *isn't* autorequired (and the service isn't declared, so that won't be required either), so it's probably working by accident. Good catch, we should definitely autorequire the package as well.